webapp/app/[locale]/tunnel/_components/evmDeposit.tsx

```diff
@@ -113,6 +127,24 @@ export const EvmDeposit = function ({ state }: EvmDepositProps) {
     !operatesNativeToken,
   )
 
+  const l1StandardBridgeAddress = getTunnelContracts(
+    hemi,
+    fromToken.chainId,
+  ).L1StandardBridge
+
+  const { isLoading: isAllowanceLoading } = useAllowance(
+    fromToken.address as `0x${string}`,
+    {
+      args: {
+        owner: address,
+        spender: l1StandardBridgeAddress,
+      },
+      query: {
+        enabled: !isNativeAddress(address),
```

Comment:
```suggestion
        enabled: !isNativeAddress(fromToken.address),
```

`address` is the user's address from the Wallet

Reply:
Resolved